directories.json

```diff
@@ -0,0 +1 @@
+["hac-maps/"]
```

Comment:
Now that I see it, we should probably do just:
["hac-maps"]
And leave the os module concatenate the directory separators.

Reply:
yup, that is valid too.
if we do that we have to change the code inside. these are very small changes

Reply:
done :)

Reply:
Thanks man, you're awesome!